testing/src/test/java/org/motechproject/nms/testing/it/ma/MobileAcademyServiceBundleIT.java

```diff
@@ -409,6 +409,31 @@ public void testNotification() {
         // TODO: cannot check the notification status yet since we don't have a real IMI url to hit
     }
 
+    @Test
+    public void testNotificationNoLocation() {
+        // Setup language/location and flw for notification
+        FrontLineWorker flw = frontLineWorkerService.getByContactNumber(2111113333L);
+        if (flw != null) {
+            flw.setStatus(FrontLineWorkerStatus.INVALID);
+            frontLineWorkerService.update(flw);
+            frontLineWorkerService.delete(flw);
+        }
+
+        flw = new FrontLineWorker("Test Worker", 2111113333L);
+        frontLineWorkerService.add(flw);
+        flw = frontLineWorkerService.getByContactNumber(2111113333L);
+        assertNotNull(flw);
+
+        long callingNumber = 2111113333L;
```

Comment:
If you're using 2111113333L everywhere and you're using a variable, why not use that same variable in the entire test instead of using the literal? Or, am I missing something?